plugins/modules/dcnm_policy.py

```diff
@@ -365,7 +367,7 @@ class DcnmPolicy:
             "POLICY_DEPLOY": "/rest/control/policies/deploy",
             "POLICY_CFG_DEPLOY": "/rest/control/fabrics/{}/config-deploy/",
             "POLICY_WITH_POLICY_ID": "/rest/control/policies/{}",
-            "CONFIG_PREVIEW": "/rest/control/fabrics/{}/config-preview?forceShowRun=false&showBrief=true",
+            "CONFIG_PREVIEW": "/rest/control/fabrics/{}/config-preview/",
```

Comment:
What is the functional difference between these two?

Reply:
The removed one was for getting all preview of all switches in the fabric which may not be scalable. So we now moved to the new one, where we specify switch serial numbers which are of interest. So the string "forceShowRun=false&showBrief=true" should come after the switch serial numbers. So we moved adding this to routine where we add switch info to the API